src/init/consented/mobile-sticky-frontend-mobile.ts

```diff
@@ -0,0 +1,49 @@
+import { createAdSlot, wrapSlotInContainer } from 'core/create-ad-slot';
+import fastdom from '../../lib/fastdom-promise';
+
+const insertCrosswordsAd = (anchor: HTMLElement) => {
+	const testSlot = createAdSlot('mobile-sticky');
```

Comment:
Shall we create a new slot name for this, instead of repurposing the `mobile-sticky`. I know the sizing will be the same, but at present the name doesn't reflect the purpose.